Makefile

```diff
@@ -139,7 +139,6 @@ clean: container-clean
 	-examples/kubernetes/stop.sh
 
 docs-lint:
-	docker run --rm -i hadolint/hadolint < Dockerfile.docs
 	tools/asciidoc-lint.sh
 
 docs:
```

Comment:
Can/should we remove this target entirely?

Reply:
Possibly however it is useful sometimes to just lint docs if only docs have changed. Initially it was for everything docs related but you're right it could be consolidated.

Reply:
I meant the `docs` target which currently does nothing (the GH UI is confusing :D )